test/topology_custom/test_hints.py

```diff
@@ -49,3 +50,61 @@ def get_hints_written_count(server):
     # Verify hints are written
     hints_after = get_hints_written_count(servers[0])
     assert hints_after > hints_before
+
+@pytest.mark.asyncio
+async def test_sync_point(manager: ManagerClient):
+    """
+    We want to verify
+    """
+    node_count = 3
+    [node1, node2, node3] = await manager.servers_add(node_count)
+
+    cql = manager.get_cql()
+    await cql.run_async("CREATE KEYSPACE ks WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3}")
+    await cql.run_async("CREATE TABLE ks.t (pk int primary key, v int)")
+
+    await manager.server_stop_gracefully(node2.server_id)
+    await manager.server_stop_gracefully(node3.server_id)
+
+    await manager.server_not_sees_other_server(node1.ip_addr, node2.ip_addr)
+    await manager.server_not_sees_other_server(node1.ip_addr, node3.ip_addr)
+
+    async def save_hints(primary_key: int) -> None:
+        await cql.run_async(SimpleStatement(f"INSERT INTO ks.t (pk, v) VALUES ({primary_key}, {primary_key})", consistency_level=ConsistencyLevel.ONE))
```

Comment:
Do we really need this helper function...? It performs a write; saving hints can only happen as a part of it, thus the name `save_hints` is a bit strange. Moreover, hints will only be written if one of the replicas is down, so it has a precondition that is not documented.

I think it's better to just inline this function to the only place where it is used.